src/DataLayer/converters/tests/MeasureConverter.spec.ts

```diff
@@ -114,4 +115,15 @@ describe('convertMeasure', () => {
             ...afm.arithmeticMeasure
         });
     });
+
+    describe('getFormat', () => {
+        it('should return default format for arithmetic measure sum operation', () => {
+            expect(MeasureConverter.getFormat(measures.arithmeticMeasure)).toBeUndefined();
+        });
+
+        it('should return percentage format for change operation', () => {
+            const changeAM = set(measures.arithmeticMeasure, 'measure.definition.arithmeticMeasure.operator', 'change');
```

Comment:
please do not use `lodash/set` I would rather see mocked objects. :) 

reason is that lodash/set is not providing good type-safety... meaning if you change  arithmeticMeasure definition due to some refactor then `measure.definition.arithmeticMeasure.operator` will be not found.